src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentNavigationService.cs

```diff
@@ -176,14 +176,16 @@ public bool TryNavigateToPosition(Workspace workspace, DocumentId documentId, in
             return NavigateTo(textBuffer, vsTextSpan);
         }
 
-        private static Document OpenDocument(Workspace workspace, DocumentId documentId, bool usePreviewTab)
+        private static Document OpenDocument(Workspace workspace, DocumentId documentId, OptionSet options)
```

Comment:
I didn't make the parameter optional because every call site passes an argument to it. It's just that those arguments could be null, and that's a valid situation.